### PR TITLE
Musicmaker

### PR DESCRIPTION
**Pull request by itcomesinwaves**

### CODE REVIEW

**Review by iamdestinos — APPROVED** [review]
Nice work on the README. Approved

File: client/src/routes/MusicMaker/Octave.tsx [hunk 1]
```diff
@@ -1,176 +1,173 @@
...
+              onClick={() => keyBoardPress("A#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("B4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("C5")}
+            ></div>
```
Comment (copelka) [inline client/src/routes/MusicMaker/Octave.tsx:165]: this repetition defeats the purpose of React. how can you write this to be DRY (Don't Repeat Yourself)?

**Review by copelka — CHANGES_REQUESTED** [review]

File: client/src/routes/MusicMaker/Octave.tsx (hunk 1 above)
Comment (itcomesinwaves) [reply client/src/routes/MusicMaker/Octave.tsx]: done!

**Review by erincwilliams0 — APPROVED** [review]
much DRY very wow

**Pull request merged**